common/country_types/hfe_country_types.txt

```diff
@@ -115,84 +122,117 @@ human_fallen_empire = {
 		}
 	}
 	modules = {
-		all_technology_module = {}
-		# tiered_technology_module = {
-		# 	tier = 5
-		# 	rare = yes
-		# }
+#		all_technology_module = {}
+		tiered_technology_module = {
+			tier = 5
+			rare = yes
+		}
+		basic_technology_module = {}
 		standard_event_module = {}
 		standard_leader_module = {}
 		standard_economy_module = {}
 		standard_species_rights_module = {}
+		
 		exclusive_diplomacy_module = {
 			can_receive = {
 				action_declare_war
+				action_offer_peace_diplomacy_view
 				action_offer_peace
 				action_make_rival
 				action_end_rivalry
 				action_offer_trade_deal
 				action_cancel_trade_deal
 				action_insult
+				action_support_independence
 			}
 			can_send = {
 				action_declare_war
+				action_offer_peace
+				action_offer_peace_diplomacy_view
 				action_make_rival
 				action_end_rivalry
 				action_offer_peace
 				action_offer_trade_deal
 				action_cancel_trade_deal
 				action_open_borders
 				action_close_borders
+				action_insult
+				action_support_independence
+#				action_ask_to_become_testing_arry = yes
 			}
 		}
 	}
+	resources = {
+		category = country_base
+		produces = {
+			energy = 100
+			minerals = 100
+			food = 100
+			alloys = 100
+			consumer_goods = 100
+			physics_research = 100
+			society_research = 100
+			engineering_research = 100
+			influence = 10
+			unity = 5
+		}
+	}
 }
 
 hfe_drone = {
 	has_difficulty_bonuses = yes
 	counts_for_victory = no
-	relations = yes # без этого не строит корабли лул
+	relations = yes # ??? ????? ?? ?????? ??????? ???
 	observable = yes
-	needs_survey = yes
-
+	needs_survey = no
+	has_complete_sensor = yes
 	playable = no
 	destroys_starbases = yes
 	share_survey_data = no
-	is_space_critter = yes
-	blocks_piracy = yes
+	is_space_critter = yes # This fixes a bug where the Drones can capture Habtiable Planets and have a primary ruling species
+	can_own_empty_planets = no
+	blocks_piracy = no
 	government = no
 	has_capital = no
 	enforces_borders = no
-	share_communications = no
-	can_have_their_comms_shared = no
+	orbital_bombardment = yes
+	attack_unarmed_orbital_stations = yes
+	share_communications = yes
```

Comment:
I dont see why this country type should share communications

Reply:
Sharing communications involves other nations being able to discover that nation. Having your communications shared means other nations can inform others through trade or contact. Sharing your communications allows for the opposite. The Harvesting Marvel does not share its communications by trade, but rather, by the built-in Vanilla rendition mechanics.

Reply:
And?

Reply:
That's my argument.

Reply:
Drone country have no diplomacy and communications, so it cannot share communications.

Reply:
Sharing communications also involves a third party. Enabling both allows you to share it, it to share itself (hypothetically), and for others to share it with you.

Reply:
And? I do not see why drones should share commnications.
They probably cant do it because of day zero contact flag.

Reply:
Actually, Zero Contact does allow it to be shared for nations that have never contacted it themselves.

Reply:
anyway I do not see why always hostile non sapient machines should share communications.

Reply:
Setting both allows all parties to share it.